src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/code/Architecture.java

```diff
@@ -126,9 +126,14 @@ public PlatformKind getWordKind() {
         return wordKind;
     }
 
-    /**
-     * Gets the name of this architecture.
-     */
+    /// Gets the name of this architecture. The value returned for
```

Comment:
This seems like a good opportunity to use the new [markdown support for javadoc](https://openjdk.org/jeps/467).